common/src/main/scala/se/lu/nateko/cp/doi/meta/DoiMeta.scala

```diff
@@ -335,6 +335,36 @@ case class Rights(rights: String, rightsUri: Option[String]) extends SelfValidat
 	)
 }
 
+enum RelationType {
+	case HasMetadata, IsMetadataFor, IsCitedBy, Cites, IsSupplementTo, IsSupplementedBy, IsContinuedBy, Continues,
+	IsDescribedBy, Describes, HasVersion, IsVersionOf, IsNewVersionOf, IsPreviousVersionOf, IsPartOf, HasPart,
+	IsPublishedIn, IsReferencedBy, References, IsDocumentedBy, Documents, IsCompiledBy, Compiles, IsVariantFormOf,
+	IsOriginalFormOf, IsIdenticalTo, IsReviewedBy, Reviews, IsDerivedFrom, IsSourceOf, IsRequiredBy, Requires,
+	IsObsoletedBy, Obsoletes, IsCollectedBy, Collects, IsTranslationOf, HasTranslation
+}
+
+enum RelatedIdentifierType {
+	case ARK, arXiv, bibcode, CSTR, DOI, EAN13, EISSN, Handle, IGSN, ISBN, ISSN, ISTC, LISSN, LSID, PMID, PURL,
```

Comment:
I'm not sure if it was necessary to list all types. Most of them will never be used and might only lead to confusion, errors, or at least scroll and scan a longer list.